src/resolvers/profileResolver.ts

```diff
@@ -32,7 +32,10 @@ const profileResolvers: any = {
             const roles = await UserRole.find({})
             return roles
         },
-
+        async getOrganizations() {
```

Comment:
who is allowed to get the organizations actually?

For secure implementation:
Unless the user is `super-admin` you have to find the associated organizations